components/compliance-service/ingest/server/compliance.go

```diff
@@ -197,14 +203,90 @@ func (s *ComplianceIngestServer) handleNotifications(ctx context.Context, report
 	return nil
 }
 
-func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata) error {
+func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata, profiles []*ingest_inspec.Profile) error {
 	if s.mgrClient == nil {
 		return fmt.Errorf("no manager client found")
 	}
+	node.ScanData.Status = getReportStatus(profiles)
 	logrus.Debugf("sendNodeInfoToManager handing-over node to manager %+v", s)
 	_, err := s.mgrClient.ProcessNode(ctx, &node)
 	if err != nil {
 		return errors.Wrap(err, "sendNodeInfoToManager error calling ProcessNode")
 	}
 	return nil
 }
+
+func getReportStatus(profiles []*ingest_inspec.Profile) nodes.LastContactData_Status {
+	// start with a status of passed as the default status
+	status := nodes.LastContactData_PASSED
+	skippedCounter := 0
+	for _, profile := range profiles {
+		profile.Status = getProfileStatus(profile)
```

Comment:
This line is confusing me. This is updating the profiles passed in statuses because they are pointers. Do we need to update the status on these profiles?
If we do, line 228 is breaking out of the for loop and will not update statuses after a failed one. Is this desired?

Reply:
if any profile status is failed, the report (node) has a failed status
if all are skipped, report (node) is skipped
otherwise passed

Reply:
Could we change the `profile.Status` to use a local variable like below? 
```suggestion
		profileStatus := getProfileStatus(profile)
```

Reply:
👍